lib/linkerd2-trace-context/src/layer.rs

```diff
@@ -99,9 +100,9 @@ impl<F: Future, S> Future for MakeFuture<F, S> {
 
 // === impl Service ===
 
-impl<Svc, B, S> tower::Service<http::Request<B>> for Service<Svc, S>
+impl<Svc, B1, B2, S> tower::Service<http::Request<B1>> for Service<Svc, S>
```

Comment:
nit: What do we in other modules that have req and rsp body types? I believe it's typical to use `A` for the request body type.

Reply:
It's inconsistent.  We have 11 files where we use `Request<A>` and 5 files where we use `Request<B1>`.  I find it pretty weird that when `B` is used alone it means "Body" but when it's used with `A` it means "the second letter of the alphabet", which is why I prefer `B1` `B2`.  Or `ReqBody` `RspBody`.

Reply:
conversely, it's common to find `T` as a base generic with `S` and `U` in relation to it, but it's a total nit. fine with this too.